login_KA.feature

```diff
@@ -0,0 +1,144 @@
+Feature: log in Scoring 
+Background: 
+    Given installed and launched VPN on local PC
+    And a Gmail account
+    And user open https://qa.ddso-spot.quantori.com page
+    And user clicks "<signed in with google>" button
+
+Scenario: 1_1.1 Log in with valid login and password which were saved
+   Given user has login and password for gmail account saved on the device
+   When user clicks on this gmail account
+   Then user is logged in
+   And user sees the main page
+
+   Examples:
```

Comment:
I think this part should have been added just as a comment to the script because it's supposed to be a part of 'Scenario Outline' and the steps have to contain '<>' to fill in. In this scenario it just won't work. And the same in the next scenarios

Reply:
Thank you! I think I misunderstood meaning of <>